ros_ws/src/control/src/control/remote_control.cpp

```diff
@@ -16,6 +16,7 @@ RemoteControl::RemoteControl()
     cmd_vel.angular.x = 0;
     cmd_vel.angular.y = 0;
     cmd_vel.angular.z = 0;
+    speedModifier = 0;
```

Comment:
We will be using underscores rather than camel case.  Here is the coding standard that we are trying to follow: https://github.com/MST-Robotics/IGVC/wiki/ROS-C---Style-Guidelines.  So for this variable it should be speed_modifier.